src/Controllers/SubmitControllerCreate.php

```diff
@@ -88,4 +119,78 @@ function ($value)
 
 		return true;
 	}
+
+	/**
+	 * Check the CMS Version
+	 *
+	 * @return  false on failiure else the CMS Version
+	 *
+	 * @since   1.0
+	 */
+	private function checkCMSVersion($data)
+	{
+		$dotchecks = explode('.', $data);
```

Comment:
See my comment on the PR. We don't need this if the preg_match function does that job for us